cves/kernel/CVE-2013-0290.yml

```
@@ -147,10 +148,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    The vulnerability was discovered by a developer, 
+    Tommi Rantala <tt.rantala@gmail.com>, testing the code with a fuzzer
```

I think this could be refined a bit, consider:

```suggestion
  answer: |
    The vulnerability was discovered by Tommi Rantala <tt.rantala@gmail.com>
    by testing the code using Trinity, a Linux system call fuzzer.
```

I am assuming the trinity in the commit https://github.com/torvalds/linux/commit/3f518bf745cbd6007d8069100fb9cb09e960c872 message is this: https://github.com/kernelslacker/trinity